packages/algorand-js/src/lib/parse-data.ts

```diff
@@ -23,8 +31,70 @@ export function convertToBuffer (s: string, encoding?: EncodingType): Buffer {
     case EncodingType.HEX: {
       return Buffer.from(s, 'hex');
     }
+    case EncodingType.STRING: {
```

Comment:
`STRING` does not represent an encoding type. Did you mean `UTF8`? If yes, then please change to utf8. (`Buffer.from(s)` defaults to utf8 only, which is the default case of switch)

Reply:
yes `UTF8`